notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Is it possible to avoid calling a "private" method like <code>.__next__()</code>?</p><p> </p><p>What about? <code>data_batch = next(ds_iter)</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done.</p>